java/test/jmri/jmrit/display/AbstractEditorTestBase.java

```diff
@@ -54,7 +54,7 @@ public void checkHelpMenuExists() {
         EditorFrameOperator jfo = new EditorFrameOperator(e);
         JMenuOperator jmo = new JMenuOperator(jfo, Bundle.getMessage("MenuHelp"));
         Assert.assertNotNull("Help Menu Exists", jmo);
-        Assert.assertEquals("Menu Item Count", 8, jmo.getItemCount());
+        Assert.assertEquals("Menu Item Count", 10, jmo.getItemCount());
```

Comment:
Did you change something in the help menu?  That total shouldn't have changed since this was created ( within the last two weeks ).

Also, if it is just failing for one of the 4 editors, then there is a problem.  The help menus should all be the same